include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -249,30 +269,27 @@ class MeasuredValues {
     // not relevant for the main calculation, as extra data for load_gen/source calculation
     std::vector<SensorCalcParam<sym>> extra_value_;
     // array of partial injection measurement, the bus with not all connected appliances measured
-    std::vector<SensorCalcParam<sym>> partial_injection_;
+    std::vector<SensorCalcParam<sym>> partial_load_injection_;
 
     // indexing array of the entries
-    // for -1 (non bus injection): connected, but no measurement
-    // for -2 (non bus injection): not connected
+    // for UNMEASURED (non bus injection): connected, but no measurement
+    // for DISCONNECTED (non bus injection): not connected
     // for <0 (bus injection): the number of unmeasured appliances in negative
-    // 	   -2 means 2 appliances are unmeasured, bus-appliance is under-determined
     //     -1 means 1 appliance is unmeasured, bus-appliance is exactly determined
+    // 	   -2 means 2 appliances are unmeasured, bus-appliance is under-determined
```

Comment:
This comment block needs to be changed since you use the new values. And you don't use negative number for unmeasured appliance anymore.